diff --git a/test/features/topics.feature b/test/features/topics.feature
index 955abe2..793217b 100644
--- a/test/features/topics.feature
+++ b/test/features/topics.feature
@@ -20,13 +20,13 @@ Feature: Topics
     And I should see "Topic2 & $p@"
     And I should see "icon" in the ".font-icon-select-1-e904" element
 
-  @api @Topics
+  @api @Topics @no-main-menu
   Scenario: See topic in the main menu
     When I am on the homepage
     Then I click on the text "Topics"
     Then I should see "Topic1"
 
-  @api @Topics
+  @api @Topics @no-main-menu
   Scenario: Check topic facet link
     When I am on the homepage
     Then I click on the text "Topics"

